packages/wonder-blocks-popover/src/components/__tests__/popover.test.tsx

```diff
@@ -356,4 +356,191 @@ describe("Popover", () => {
             ).toBeInTheDocument();
         });
     });
+
+    describe("keyboard navigation", () => {
+        it("should move focus to the first focusable element after popover is open", async () => {
+            // Arrange
+            render(
+                <>
+                    <Button>Prev focusable element outside</Button>
+                    <Popover
+                        onClose={jest.fn()}
+                        content={
+                            <PopoverContent
+                                title="Popover title"
+                                content="content"
+                                actions={<Button>Button inside popover</Button>}
+                            />
```

Comment:
**question:** are there multiple focusable items in this popover content?  If not, could you add another button?